src/Repository/TalkRepositoryInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types=1);
+
+namespace App\Repository;
+
+use App\Dto\TalkRequest;
+use App\Entity\Talk;
+use Ramsey\Uuid\UuidInterface;
+
+interface TalkRepositoryInterface
```

Comment:
Too massive for an interface, could you do smaller interfaces and make this one extend the others (ie. `TalkFactoryInterface`)

Reply:
IMO this interface contains the perfect number of function